lang/java/reef-common/src/main/java/org/apache/reef/runtime/common/driver/client/LoggingJobStatusHandler.java

```diff
@@ -28,7 +28,9 @@
 /**
  * A handler for job status messages that just logs them.
  */
-final class LoggingJobStatusHandler implements EventHandler<ReefServiceProtos.JobStatusProto> {
+@DriverSide
+final class LoggingJobStatusHandler implements JobStatusHandler {
```

Comment:
I think these changes to the client interface could go to another PR, as they are orthogonal to the other changes here.